CONTRIBUTING.md

```diff
@@ -56,27 +56,27 @@ Caused by: missing field `package`
 If desired, you can download a WebDriver manually. For instructions, please consult the documentation of the WebDriver you wish to install.
 
 You might want to pass the argument `--mode no-install` to `wasm-pack` to make sure it doesn't try to install something.
-If the WebDriver binary is in the path you can use the same command as above to run the tests.
+If the WebDriver binary is in the path, you can use the same command as above to run the tests.
 If not, you need to tell `wasm-pack test` where to find it using the `--geckodriver`, `--chromedriver`, or `--safaridriver` option.
 Run `wasm-pack test --help` to learn more.
 
 #### Running all tests
 
-You can use the `/ci/run_tests.sh` script to run all tests. This is the same script that is used by Yew's CI to run the whole test suite.
+You can use the `/ci/run_tests.sh` script to run all tests, which is the same script that is used by Yew's CI to run the whole test suite.
 The script currently always runs the tests in Firefox.
 
 #### Fetch service tests
 
-This only applies to the `yew` (and `yew-stdweb`) crate.
+That only applies to the `yew` (and `yew-stdweb`) crate.
```

Comment:
"This" is referring to the current section. What about this:
```suggestion
The following only applies to the `yew` (and `yew-stdweb`) crate.
```

Reply:
Sounds good!